tests/system/node/2/config.yaml

```diff
@@ -15,6 +15,7 @@ node:
   data_dir: .cache
 
   validator_cycle: 20
+  block_interval_sec: 5
```

Comment:
If this is added as an item of a config, we need to add it to `NodeConfig` structure and parse it in `parseNodeConfig` function.

Reply:
It already was in [NodeConfig](https://github.com/bpfkorea/agora/blob/c0f000595de29fbe8ce837f7e0eb8a88e3d10c10/source/agora/common/Config.d#L370) we were just not using it in these configs.

Reply:
> It already was in NodeConfig we were just not using it in these configs.

Sorry, I didn't see it.